runtime/near-vm-logic/src/logic.rs

```diff
@@ -2673,6 +2675,12 @@ impl<'a> VMLogic<'a> {
         }
     }
 
+    /// Add a cost for loading the contract code in the VM.
+    ///
+    /// This cost is "dumb", in that it does not consider the structure of the
+    /// contract code, only the size. This is currently the only loading fee.
+    /// A smarter fee could be added, although that would have to happen slightly
+    /// later, as this dumb fee can be pre-charged without looking at the code.
```

Comment:
Later, as in after the loading. Because we need the information from the loading phase to know the structure.

I rephrased it, I hope it is clearer now.